### PR TITLE
Waybar bugfix and minor QOL change

### PR DESCRIPTION
**Pull request by boulder-roller**
# Pull Request

### waybar toggle QOL
This was bugging me when i was changing my waybar layout. the toggle waybar keybind now reloads the config.jsonc file when being toggled on

### Waybar notification module fix
Upstream change to waybar broke this module, minor change in format string fixes it.

## Type of change

- [x] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected)
- [ ] **Documentation update** (non-breaking change; modified files are limited to the documentations)
- [ ] **Technical debt** (a code change that does not fix a bug or add a feature but makes something clearer for devs)
- [ ] **Other** (provide details below)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/prasanthrangan/hyprdots/blob/main/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [x] My commit message follows the [commit guidelines](https://github.com/prasanthrangan/hyprdots/blob/main/CONTRIBUTING.md#git-commit-messages).
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added a changelog entry.
- [ ] I have added necessary comments/documentation to my code.
- [ ] I have added tests to cover my changes.
- [ ] I have tested my code locally and it works as expected.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: Configs/.config/hypr/keybindings.conf [hunk 1]
```diff
@@ -25,7 +25,8 @@ bind = Alt, Return, fullscreen, # toggle the window between focus and fullscreen
 bind = $mainMod, L, exec, swaylock # launch lock screen
 bind = $mainMod+Shift, F, exec, $scrPath/windowpin.sh # toggle pin on focused window
 bind = $mainMod, Backspace, exec, $scrPath/logoutlaunch.sh # launch logout menu
-bind = Ctrl+Alt, W, exec, killall waybar || waybar # toggle waybar
+bind = Ctrl+Alt, W, exec, killall waybar || (wbarconfgen.sh && waybar) # toggle waybar and reload config
```
Comment (kRHYME7) [inline Configs/.config/hypr/keybindings.conf:28]: Should be ` reload_flag=1   wbarconfgen.sh ` 

or `env reload_flag=1   wbarconfgen.sh ` 

just try which works

File: Configs/.config/waybar/modules/notifications.jsonc [hunk 2]
```diff
@@ -1,5 +1,5 @@
     "custom/notifications": {
-        "format": "{icon} {}",
+        "format": "{} {icon}",
```
Comment (kRHYME7) [inline Configs/.config/waybar/modules/notifications.jsonc:2]: I don't understand the change. moving the icon to the right?

**Review by kRHYME7 — CHANGES_REQUESTED** [review]

File: Configs/.config/waybar/modules/notifications.jsonc (hunk 2 above)
Comment (boulder-roller) [reply Configs/.config/waybar/modules/notifications.jsonc:2]: as i understand it something upstream with waybar causes the module to break and not display unless the icon is after the number.... yeah idk why, but a numerous people said this fix works so :shrug:

File: Configs/.config/hypr/keybindings.conf [hunk 3]
```diff
@@ -25,7 +25,7 @@ bind = Alt, Return, fullscreen, # toggle the window between focus and fullscreen
 bind = $mainMod, L, exec, swaylock # launch lock screen
 bind = $mainMod+Shift, F, exec, $scrPath/windowpin.sh # toggle pin on focused window
 bind = $mainMod, Backspace, exec, $scrPath/logoutlaunch.sh # launch logout menu
-bind = Ctrl+Alt, W, exec, killall waybar || (wbarconfgen.sh && waybar) # toggle waybar and reload config
+bind = Ctrl+Alt, W, exec, killall waybar || (env reload_flag=1 wbarconfgen.sh) # toggle waybar and reload config
```
Comment (kRHYME7) [inline Configs/.config/hypr/keybindings.conf:28]: My bad. Let’s stick with the conventions. 


``` hyprlang
bind = Ctrl+Alt, W, exec, killall waybar || (env reload_flag=1 $scrPath/wbarconfgen.sh) # toggle waybar and reload config
```

**Review by kRHYME7 — CHANGES_REQUESTED** [review]

File: Configs/.config/waybar/modules/notifications.jsonc (hunk 2 above)
Comment (kRHYME7) [reply Configs/.config/waybar/modules/notifications.jsonc:2]: can you help me find that upstream thread?

Comment (boulder-roller) [reply Configs/.config/waybar/modules/notifications.jsonc:2]: https://github.com/Alexays/Waybar/issues/3623

File: Configs/.config/hypr/keybindings.conf (hunk 3 above)
Comment (boulder-roller) [reply Configs/.config/hypr/keybindings.conf]: should be good now

**Comment by kRHYME7** [comment]
Thanks will update this later

**Review by kRHYME7 — APPROVED** [review]
Please Review first  @boulder-roller

**Review by rishav12s — APPROVED** [review]

**Comment by rishav12s** [comment]
dont have multiple monitors so im not sure about that 

tried setting it DP-1 and it hides the waybar on my laptop display eDP-1 so ig its working fine

notification works fine no errors thrown when running from terminal

**Comment by kRHYME7** [comment]
# This PR added and option to have the waybar output set in hyde.conf

**Pull request merged**